#!/bin/sh /usr/share/dpatch/dpatch-run
## 325-magic-add-subversion.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: New magic for SVN dumps (closes: #256652).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/revision file-4.21/magic/Magdir/revision
--- file-4.21.orig/magic/Magdir/revision	2007-04-09 16:33:07.000000000 +0000
+++ file-4.21/magic/Magdir/revision	2007-06-01 18:52:52.000000000 +0000
@@ -7,3 +7,8 @@
 # Conary changesets
 # From: Jonathan Smith <smithj@rpath.com>
 0	belong	0xea3f81bb	Conary changeset data
+
+# Type: Subversion (SVN) dumps
+# From: Uwe Zeisberger <zeisberg@informatik.uni-freiburg.de>
+0	string	SVN-fs-dump-format-version:	Subversion dumpfile
+>28	string	>\0				(version: %s)
